openbb_terminal/stocks/options/options_controller.py

```diff
@@ -814,20 +830,17 @@ def call_chains(self, other_args: List[str]):
         if ns_parser:
             if self.ticker:
                 if self.selected_date:
-                    if ns_parser.source == "Tradier" or self.source == "Tradier":
-                        if API_TRADIER_TOKEN != "REPLACE_ME":  # nosec
-                            tradier_view.display_chains(
-                                symbol=self.ticker,
-                                expiry=self.selected_date,
-                                to_display=ns_parser.to_display,
-                                min_sp=ns_parser.min_sp,
-                                max_sp=ns_parser.max_sp,
-                                calls_only=ns_parser.calls,
-                                puts_only=ns_parser.puts,
-                                export=ns_parser.export,
-                            )
-                        else:
-                            console.print("TRADIER TOKEN not supplied. \n")
+                    if ns_parser.source == "Tradier":
```

Comment:
Since we already have the option chain loaded, shouldn't chains be source independent and similar to the vol/oi/voi commands that use the loaded df and allow to specify exp ?